src/lib/wicked/adapter.js

```diff
@@ -113,6 +113,35 @@ class WickedAdapter {
         return result;
     }
 
+
+    /*
+     * Obtains the DNS global settings from /etc/sysconfig/network/config
+     */
+    async dnsSettings() {
+        const filePath = `/etc/sysconfig/network/config`;
+        const file = await new SysconfigFile(filePath).read();
+        const policy = await file.get("NETCONFIG_DNS_POLICY") || "";
+        const nameServers = await file.get("NETCONFIG_DNS_STATIC_SERVERS") || "";
```

Comment:
Using this approach is OK. But perhaps you would like to subclass `SysconfigFile` and do some further processing:

```js
class DnsConfig extends SysconfigFile { // OK, I suck at naming things... :)
    dnsStaticServers() {
        const this.get('NETCONFIG_DNS_STATIC_SERVERS', "").split(" ")
    }
}
```

Or just:

```js
class DnsConfig extends SysconfigFile {
    get(key) {
        const value = super.get(key, "");

        // or define a list of fields that are arrays...
        // or keep a registry of keys and types (so you can transform them accordingly)...
        // you know what I mean :)
        if (key === 'NETCONFIG_DNS_STATIC_SERVERS') {
            return value ? value.split(" ") : [];
        }

        return value;
    }
}
```